Add gradle-wrapper.jar

valentingoebel opened the pull request:
I saw https://github.com/jeffwils/grails-spring-security-saml/issues/18 and then I realised that the wrapper was missing.

 I apologise that I accidentially didn't add the gradle wrapper because of the .gitignore rule.

Please merge this change after testing it.

irstevenson commented:
Heh, I was actually working on adding the wrappers when I found #18. And unfortunately what I learnt there is: that grails **does not** use the gradle wrapper in the project - even when launched with the grails wrapper. Very frustrating - and at first confusing.

None the less though, I believe this definitely needs to be done, so I tested this branch and the wrapper is working. However, could you also bring in the grails wrapper. For that you need copy the following from a fresh 3.3.0 app (i.e. `grails create-app`):

* `grailsw`
* `grailsw.bat`
* `grails-wrapper.bat`

(I was almost going to do it direct on this branch, but wasn't sure if that'd be okay.)

valentingoebel commented:
Interesting! I accidentially made that same mistake.

@irstevenson irstevenson requested review from irstevenson and removed request for irstevenson 6 hours ago

@valentingoebel valentingoebel requested review from jeffwils and removed request for jeffwils 4 minutes ago

I've added the grails wrapper.

irstevenson commented:
Rebuilt and cleaned via wrappers (grailsw and gradlew) and all appears to still work.

Also note the removal of the message from #21.

GTG!
